### ISSUE TITLE
1.2.7 _Push User sometimes hosts DHIS2 usernames, other times DHIS2 user's Name and Surname

### ISSUE THREAD

**Issue by cristinaelle** [EyeSeeTea/QAApp#202]
ifoche_test: https://android.psi-mis.org/api/events/E6IrRCEeKGx

Ignacio Foche: https://android.psi-mis.org/api/events/YpKkBRKZntD

cristina

**Comment by ifoche** [comment]
Hi Cristina. As far as I see, looks like what you're seeing there must be just the consequence of using the same server for dev and test :). As we didn't have any staging server to deploy the new features to, we used android server, which we use mostly during the development and seems that the first event is completed by ifoche_test user, while the second one is by KEDemo1. If I had to guess I would say that multiple users using the same OU to upload data has confused us, and you're looking at 2 different surveys from 2 different users.
